test/fn/csv-to-docs.spec.js

```diff
@@ -36,7 +36,7 @@ describe('csv-to-docs', function() {
             const expectedDocsDir  = `${dir}/expected-json_docs`;
 
             // then
```

Comment:
Okay, we have great tests for the internals of the `sync-fs.warnIfDirectoryIsNotEmpty` function. The last piece to have total test coverage is just to validate that the function is actually called during the `execute`.  Should be able to stub the function and just assert here that it was called with the expected warning message.